s/features/notes/components/notes-indicator/xiome-notes-indicator.css.ts

```diff
@@ -0,0 +1,37 @@
+
+import {css} from "../../../../framework/component.js"
+export default css`
+
+:host {
+	display: inline-block;
+	--xiome-notes-indicator-size: 1em;
+	--xiome-notes-indicator-color: red;
+	--xiome-notes-indicator-bleedout: 0em;
+	width: 3em;
+	height: 3em;
+	position: relative;
+}
+
+:host([no-icon]) .bell {
+	display: none;
+}
+
+xiome-notes-indicator {
```

Comment:
- this is broken.  
  - setting the --xiome-notes-indicator-size from the outside (on the mocksite page) doesn't have any affect on the size.
  - this selector `xiome-notes-indicator` doesn't work within this context
- please test whether the other custom properties are working too